integrationtest/neo/node/request/GetAll.d

```diff
@@ -38,15 +38,23 @@ public void handle ( Object shared_resources, RequestOnConn connection,
     auto storage = cast(Storage)shared_resources;
     assert(storage);
 
+    auto ed = connection.event_dispatcher;
+
     switch ( cmdver )
     {
         case 0:
+            ed.send(
+                ( ed.Payload payload )
+                {
+                    payload.addConstant(GlobalStatusCode.RequestSupported);
+                }
+            );
```

Comment:
This ought to be flushed.

Reply:
(All the more reason for https://github.com/sociomantic-tsunami/swarm/issues/168.)